location-service/src/main/java/com/movie/recommender/location/model/entity/Country.java

```diff
@@ -0,0 +1,26 @@
+package com.movie.recommender.location.model.entity;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.util.List;
+
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "countries")
+public class Country {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    private String name;
+
+    @OneToMany
```

Comment:
add a `mappedBy` attribute in `@OneToMany` to establish a bidirectional relationship